ui/app/main/src/components/pages/page-insights.ts

```diff
@@ -53,8 +110,8 @@ const viewerConfig: DataViewerConfig = {
             gridRowStart: "1"
         },
         defaults: [{
-            assetId:"5442Ite2XVp3Ky9X2y647a",
-            attributes: ["UVIndex"]
+            assetId:"4m4dWZUAV4IepHKc8Wwv85",
```

Comment:
Not clear what these defaults mean but no good having `assetIds` hardcoded as they won't be valid between installations

Reply:
As discussed I've passed this to Don.